moon2/values.yaml

```diff
@@ -116,6 +116,16 @@ service:
   ##
   externalIPs: []
 
+
+##
+## Moon pod-deployment settings.
+## 
+pod:
```

Comment:
Let's place this near deployment annotations:

```
##
## Moon deployment settings.
##
deployment:

  ##
  ## Custom deployment annotations.
  ##
  annotations: {}

  ##
  ## Custom pod annotations.
  ##
  podAnnotations: {}
```

Reply:
Or shall it be moved to deployment? 🤔  @vania-pooh

Reply:
Think deployment makes more sense

Reply:
@Alien2150 don't forget to remove `pod:`. Currently it is still present.